src/sage/combinat/symmetric_group_algebra.py

```diff
@@ -1941,6 +1943,42 @@ def _dft_seminormal(self, mult='l2r'):
         snb = self.seminormal_basis(mult=mult)
         return matrix([vector(b) for b in snb]).inverse().transpose()
 
+    def _dft_modular(self):
+        """
+        Return the discrete Foruier transform when the characterisc divides the order of the group.
+        The usual .dft() throws ZeroDivisionError when p|n.
```

Comment:
I'd remove the reference to the `.dft()`-method here; there's also a small typo.
```suggestion
        Return the discrete Foruier transform when the characteristic divides the order of the group.
```